### ISSUE TITLE
this is not really portable

### ISSUE THREAD

**Issue by MyTDT-Mysoft** [daemondevin/7-ZipPortable#2]
it isnt portable if it uses a installer... thats insane lol (i know portable apps is doing the same thing... but its still insane... and thats NOT PORTABLE)

**Comment by daemondevin** [comment]
### TL;DR
The installer you're talking about is just a container or better known as an archive; like a zip archive only packaged for use as an executable file. With that said, I feel like I need to tell you that you're _issue_ is merely an opinion which you could have simply kept to yourself instead of disrespecting this repository with your shallow attempt to seem like you know what you're talking about; which you don't.

### TL;RA
Just in case you didn't understand the summarized version above.. I'll dumb it down for you. The installer you're referring to is just a glorified archive packaged using the [LZMA](https://en.wikipedia.org/wiki/Lempel%E2%80%93Ziv%E2%80%93Markov_chain_algorithm) which provides lossless data compression. If you're familiar with **WinRAR**, which is another file archiver, but if you're not this program has a feature to compress into a self-extracting executable file (i.e. ***.exe**). Think of it like a regular zip archive that you can just double click to execute itself by extracting the contents that are inside it into a folder of your choosing. 

I would think you should know about the aforementioned considering 7-Zip has the above mentioned capabilities as well. The fact that NSIS (which is the packaging utility used to make the portable installer as well as carry out the portable functionality for the program itself) uses the LZMA to compress the package contents for a much better compression ratio for a reduced file size allowing faster data transfer and download speeds, does not mean it's leaving trash on an end-user's machine. 

On another note, what you've stated here was a fucking **opinion**. Something that you based solely on your feelings about the matter or what you perceive to be true. However, the fact is that you're _opinion_ was not necessary and should have been kept to yourself. Next time you should remember, it is better to remain quiet and be thought a fool then to open your mouth and remove all doubt.

**Comment by MyTDT-Mysoft** [comment]
ok... and how do i "install" that from my phone? and how do i "install" the 32bit portable from 64bit windows?

**Comment by daemondevin** [comment]
This portable will launch the appropriate version of the program depending on what your running. So if you're on a 32bit machine it'll launch the 32bit version of the program; same goes for the 64bit version. 

As far as installing from your phone.. well you can't but what you can do is just download this packages contents in a zip archive and extract them to wherever you want later or whatever. Just click on the button that says **Code** on this repository's homepage and select _Download Zip_. I'm sure you can take it from there.
